### DISCUSSION TITLE
Support installation instruction for Fedora

### DISCUSSION THREAD

**Discussion by thai-nm** [BambooEngine/ibus-bamboo#354] [Ideas]
I have just switched from Ubuntu to Fedora (version 36).
In Ubuntu, I just copy-paste the command shown in the README.md file and then everything works well. But in Fedora, I tried to install with OpenBuildService in both 2 ways: RPM file and manually installing the repository. No errors, but I can't find anything or any button to switch between English and Vietnamese. Please add more instructions or help me, thanks a lot :D

**Comment by darkhatplatypus** [comment]
Hi, ibus-bamboo now shows up in the Settings app, under Keyboard > Input Sources on Fedora 38. You can switch input methods with the default keyboard shortcut `Super+Space`. 
Edit: no issue for now, the preferences works fine
![image](https://github.com/BambooEngine/ibus-bamboo/assets/9989840/9f5a66cb-1279-442b-b346-946036055e08)
